cmd/zfs/zfs_main.c

```diff
@@ -7471,16 +7471,17 @@ zfs_do_diff(int argc, char **argv)
 }
 
 /*
- * zfs bookmark <fs@snap> <fs#bmark>
+ * zfs bookmark <fs@target>|<fs#target> <fs#bookmark>
  *
- * Creates a bookmark with the given name from the given snapshot.
+ * Creates a bookmark with the given name from the target snapshot
+ * or creates a copy of an existing target bookmark.
```

Comment:
I personally don't like the use of the term "Target", "given snapshot" is clear to everyone.
"Target" me be interprented the other way around (as in: Goal), due to translation errors.